ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixParticipantTest.java

```diff
@@ -191,8 +191,10 @@ public void testGetAndSetReplicaStoppedState() throws IOException, JSONException
         new HelixParticipant(new ClusterMapConfig(new VerifiableProperties(props)), helixManagerFactory);
     HelixParticipant helixParticipantDummy =
         new HelixParticipant(new ClusterMapConfig(new VerifiableProperties(propsDummy)), helixManagerFactory);
+    HelixParticipant helixParticipantSpy = Mockito.spy(helixParticipant);
```

Comment:
Here we use Mockito spy() to check if the method is invoked or not in an object.